javascript/ql/lib/semmle/javascript/frameworks/Connect.qll

```diff
@@ -156,10 +113,11 @@ module Connect {
    * An access to a user-controlled Connect request input.
    */
   private class RequestInputAccess extends HTTP::RequestInputAccess {
-    RequestExpr request;
+    NodeJSLib::RequestExpr request;
```

Comment:
Could we keep hiding that this is just a `NodeJSLib::RequestExpr` reference under the hood?

Perhaps the deleted `RequestExpr` should just be an alias now?

Reply:
I've added a `class RequestExpr = NodeJSLib::RequestExpr;`